assets/css/styles.scss

```diff
@@ -105,8 +105,8 @@ h3 {
   }
 
   img {
-    min-width: 25em;
-    max-width: 35em;
+    min-width: 6em;
+    max-width: 36em;
```

Comment:
Why `36`? May be able to remove `min-width` altogether here; the previous assumption I think was all images were wider than the viewport.

Reply:
36 em is 6 inches (for a 12 pt font size), which is a good width for a full size image. I'm not wed to any of these numbers. What do you suggest?

Reply:
Let's leave it as 35em (that's the viewport max width), remove the `min-width` and see if that's enough?

Reply:
Done.

Reply:
Word of warning, this PR is 💯 cowboy coded and untested.

Reply:
🤠

Reply:
Cool. Mind testing it locally (or asking me to do so) and then we can merge?

Reply:
I have thesis deadlines, so I won't test myself today, could do next week though. If you're able to test, that'd be awesome.